dist/tools/flake8/check.sh

```diff
@@ -43,7 +45,19 @@ ${FLAKE8_CMD} --version &> /dev/null || {
     exit 1
 }
 
-ERRORS=$(${FLAKE8_CMD} --config="${RIOTTOOLS}"/flake8/flake8.cfg ${FILES})
+ERRORS=$(${FLAKE8_CMD} --config="${RIOTTOOLS}"/flake8/flake8.cfg ${FILES} | ${LOG})
```

Comment:
Here I would just use the `ERRORS` variable instead of indirecting via `LOG`, as I did in the [doccheck](https://github.com/RIOT-OS/RIOT/blob/5d30d7993e725f2f2335964bdf78a35b87552ee2/dist/tools/doccheck/check.sh#L41-L48)